gh_test.go

```diff
@@ -103,6 +103,32 @@ func TestGQLClient(t *testing.T) {
 	assert.Equal(t, "token GH_TOKEN", http.Requests[0].Header.Get("Authorization"))
 }
 
+func TestGQLClientError(t *testing.T) {
+	tempDir := t.TempDir()
+	orig_GH_CONFIG_DIR := os.Getenv("GH_CONFIG_DIR")
+	orig_GH_TOKEN := os.Getenv("GH_TOKEN")
+	t.Cleanup(func() {
+		os.Setenv("GH_CONFIG_DIR", orig_GH_CONFIG_DIR)
+		os.Setenv("GH_TOKEN", orig_GH_TOKEN)
+	})
+	os.Setenv("GH_CONFIG_DIR", tempDir)
+	os.Setenv("GH_TOKEN", "GH_TOKEN")
+
+	http := httpmock.NewRegistry(t)
+	http.Register(
+		httpmock.GQL("QUERY"),
+		httpmock.StringResponse(`{"errors":[{"type":"NOT_FOUND","path":["organization"],"message":"Could not resolve to an Organization with the login of 'cli'."}]}`),
+	)
```

Comment:
I just merged in the migration to `gock` so if we could change this mock to use the new approach that would be great 🙏